packages/@glimmer/runtime/lib/vm/append.ts

```diff
@@ -27,6 +27,11 @@ export interface PublicVM {
   newDestroyable(d: Destroyable): void;
 }
 
+export interface IteratorResult<T> {
```

Comment:
@chadhietala it should be coming from node_modules/typescript/lib/lib.es2015.d.ts